src/coreclr/vm/corelib.h

```diff
@@ -264,11 +264,9 @@ DEFINE_CLASS(UINT128,              System,                 UInt128)
 
 DEFINE_CLASS(MATH,                  System,                 Math)
 DEFINE_METHOD(MATH,                 ROUND,                  Round,                      SM_Dbl_RetDbl)
-DEFINE_METHOD(MATH,                 FMOD,                   FMod,                       NoSig)
 
 DEFINE_CLASS(MATHF,                 System,                 MathF)
 DEFINE_METHOD(MATHF,                ROUND,                  Round,                      SM_Flt_RetFlt)
-DEFINE_METHOD(MATHF,                FMOD,                   FMod,                       NoSig)
```

Comment:
Can we also just remove `Math.FMod`

We can just replace any calls with simply `x % y`, since that does the same thing

Reply:
> Can we also just remove `Math.FMod`
> 
> We can just replace any calls with simply `x % y`, since that does the same thing

Won't this make porting `FMod/%` to a full managed impl harder later on?

Also FMod has some special intrinsic recognition in the JIT, not sure if `%` has the same today.

Reply:
> Won't this make porting FMod/% to a full managed impl harder later on?

If FMod is ever ported to a fully managed impl, this can be adjusted as necessary.

> Also FMod has some special intrinsic recognition in the JIT, not sure if % has the same today.

`%` has same value numbering support as FMod.